doc/nrf/templates/Integration_template_README.rst

```diff
@@ -1,41 +1,41 @@
-.. _Ecosystem_integration_template:
+.. _integration_template:
 
-Template: Ecosystem integration
-###############################
+Template: Integration
+#####################
 
 .. contents::
    :local:
    :depth: 2
 
 .. note::
-   Provide a short introductory description of the tool or service (third-party), indicating the value it brings to |NCS|.
+   Provide a short introductory description of the product (third-party), indicating the value it brings to |NCS|.
    Refer to the following instructions to create the description:
 
-   * Provide a title that starts with the tool or service name with "integration" as a suffix, for example, "Memfault integration".
+   * Provide a title that starts with the product name with "integration" as a suffix, for example, "Memfault integration".
    * Place the documentation inside the :file:`nrf/doc/nrf` folder.
-   * List the file name with path in the ``.. toctree:`` of the :file:`ecosystem_integration` RST file.
+   * List the file name with path in the ``.. toctree:`` of the :file:`integrations` RST file.
    * Sections with * are optional and can be left out.
      All other sections are required.
      There could be more sections, which can be added after discussion with the tech writer team.
      However, it should have the mandatory sections from the template in the recommended format at the very least.
 
 .. tip::
-   Describe the tool or service in one or two sentences maximum (full sentences, not sentence fragments), so that the user gets a clear idea of what the tool or service does.
-   This introduction must be concise and clear, highlighting the features of the tool or service that it brings to |NCS|.
+   Describe the product in one or two sentences maximum (full sentences, not sentence fragments), so that the user gets a clear idea of what the product does.
+   This introduction must be concise and clear, highlighting the features of the product that it brings to |NCS|.
 
 Partner overview*
 *****************
 
 .. note::
-   Provide an explanation of the tool or service.
-   This is optional if the tool or service is described above.
+   Provide an explanation of the product.
+   This is optional if the product is described above.
```

Comment:
```suggestion
   This is optional if the third-party product is so simple that the introductory two-sentence description is covering all the details.
```
If we're aiming for two sentences in the short desc and we want to convey there what the product does and what it brings to the NCS, we will definitely need this section in my opinion. Two sentences in the short desc for covering everything that should also go into overview is too optimistic assumption.